### PR TITLE
Added tablet css for showcase and control-code

### PR DESCRIPTION
**Pull request by totallycrow**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -1,414 +1,658 @@
...
-/* The dots/bullets/indicators */
-.dot {
-  cursor: pointer;
-  height: 5px;
-  width: 5px;
-  margin: 0 5px;
-  background-color: #bbb;
-  border-radius: 50%;
-  display: inline-block;
+@media screen and (min-width: 320px) and (max-width: 1024px) {
```
Comment (CalogerN) [inline css/main.css]: This (max-width: 1024px) overlaps with the tablet version, we should reduce it to (max-width: 768px)

File: css/main.css [hunk 2]
```diff
@@ -1,414 +1,658 @@
...
+    margin-bottom: 100px;
+    color: #858585;
+    margin-top: 30px;
+  }
 }
 
-.active, .dot:hover {
-  background-color: #717171;
+/* TABLET View */
+@media (min-width: 768px) and (max-width: 1024px) {
```
Comment (CalogerN) [inline css/main.css]: We have to change (min-width: 769px)  or it will cause problems.

File: css/main.css [hunk 3]
```diff
@@ -1,414 +1,658 @@
...
+  /* body {
+    background-color: red;
+  } */
```
Comment (CalogerN) [inline css/main.css]: Why this comment? Are you testing if css works?

File: css/main.css [hunk 4]
```diff
@@ -1,414 +1,658 @@
...
+* {
+  margin: 0;
+  padding: 0;
 }
 
-/* Hide the images by default */
-.mySlides {
-  display: none;
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
```
Comment (CalogerN) [inline css/main.css]: You forgot to add a standard font-size or your rem units will not work as you want.

**Comment by totallycrow** [comment]
@CalogerN can you have a look on the requested changes

**Pull request merged**